examples/escrow-cli/escrow_contract.rs

```diff
@@ -104,14 +112,16 @@ fn claim<Record: TxORecord<EscrowDatum, ()>>(
 fn lookup_output<Record: TxORecord<EscrowDatum, ()>>(
     id: &str,
     txo_record: &Record,
-) -> NauResult<Output<EscrowDatum>> {
+) -> SCLogicResult<Output<EscrowDatum>> {
     let script_address = EscrowValidatorScript.address();
     let outputs = txo_record.outputs_at_address(&script_address);
     outputs
         .iter()
         .find(|o| o.id() == id)
         .cloned()
-        .ok_or(format!("Couldn't find output with id: {}", id))
+        .ok_or(SCLogicError::Lookup(
+            Box::new(TxORecordError::FailedToRetrieveOutputWithId(id.to_string()))
```

Comment:
Yeah. We don't want to initialize a `TxORecordError` outside of a `TxORecord`. I'd just create an

```
enum EscrowContractError {
    OutputNotFound(String),
    ...
}
```

These errors will capture concepts unique to the escrow contract.